ronin-web-browser.gemspec

```diff
@@ -36,26 +36,22 @@ Gem::Specification.new do |gem|
   gem.extensions       = glob[gemspec['extensions'] || 'ext/**/extconf.rb']
   gem.extra_rdoc_files = glob[gemspec['extra_doc_files'] || '*.{txt,md}']
 
-  gem.require_paths = Array(gemspec.fetch('require_paths') {
+  gem.require_paths = Array(gemspec.fetch('require_paths') do
     %w[ext lib].select { |dir| File.directory?(dir) }
-  })
+  end)
 
   gem.requirements              = gemspec['requirements']
   gem.required_ruby_version     = gemspec['required_ruby_version']
   gem.required_rubygems_version = gemspec['required_rubygems_version']
   gem.post_install_message      = gemspec['post_install_message']
 
-  split = lambda { |string| string.split(/,\s*/) }
+  split = ->(string) { string.split(/,\s*/) }
 
-  if gemspec['dependencies']
-    gemspec['dependencies'].each do |name,versions|
-      gem.add_dependency(name,split[versions])
-    end
+  gemspec['dependencies']&.each do |name, versions|
+    gem.add_dependency(name, split[versions])
   end
 
-  if gemspec['development_dependencies']
-    gemspec['development_dependencies'].each do |name,versions|
-      gem.add_development_dependency(name,split[versions])
-    end
+  gemspec['development_dependencies']&.each do |name, versions|
+    gem.add_development_dependency(name, split[versions])
```

Comment:
I would just ignore the `.gemspec` file. It's a boilerplate file that can be updated later by coping in a new version.